### PR TITLE
Add testID To TouchableWithoutFeedbackProperties

### PR DESCRIPTION
**Pull request by huhuanming**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/facebook/react-native/blob/master/Libraries/Components/Touchable/TouchableWithoutFeedback.js>>
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by gyzerok — APPROVED** [review]
LGTM

**Comment by gyzerok** [comment]
Ah, forgot 👍

**Comment by huhuanming** [comment]
thx

File: types/react-native/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Type definitions for react-native 0.44
 // Project: https://github.com/facebook/react-native
-// Definitions by: Eloy Durán <https://github.com/alloy>, Fedor Nezhivoi <https://github.com/gyzerok>
+// Definitions by: Eloy Durán <https://github.com/alloy>, Fedor Nezhivoi <https://github.com/gyzerok>, HuHuanming <https://github.com/huhuanming>
```
Comment (alloy) [inline types/react-native/index.d.ts]: This header isn't named the best. Adding yourself to this list means you want to be a maintainer and you will get notifications for each React Native related PR. Did you really intend to do that?

Comment (huhuanming) [inline types/react-native/index.d.ts]: 😳 sure. I want!

Comment (huhuanming) [inline types/react-native/index.d.ts]: I have recently added @types/react-native-google-analytics-bridge, @types/react-navigation, @types/jpush-react-native and some improves on other types repo.
I hope that I can make my contribution to the TypeScript and Developers.

Comment (alloy) [inline types/react-native/index.d.ts]: Awesome, thanks 🙌

**Pull request merged**